terracumber_config/tf_files/tfvars/PR-testing-manager43.tfvars

```diff
@@ -0,0 +1,16 @@
+############ Spacewalk unique variables ############
+
+IMAGE                  = "sles15sp4o"
+IMAGES                 = ["rocky9o", "opensuse154o", "sles15sp4o", "ubuntu2204o"]
```

Comment:
Rocky9 is correct?

Reply:
In SUSE-Manager-4.3-PR-tests-env5.tf, I can see we are using rocky9